### DISCUSSION TITLE
my all accounts fogot

### DISCUSSION THREAD

**Discussion by GoGoGaG** [Authenticator-Extension/Authenticator#863] [Q&A] [answered]
### Describe the issue

My Windows has been corrupted and I formatted the hard disk and the computer and did not take any backups because I did not expect this to happen and currently Discord does not want to return my account to me. Please help me quickly

### Browser

Chrome

### Browser Version

i did‘t know

### Extension Version

i can‘t know

**Comment by mymindstorm** [comment, marked as answer]
If you lost your backup files, then you cannot get your codes back. There isn't anything we can do about that on our end. Have a look at https://authenticator.cc/docs/en/lost-codes.
